deploy-manage/remote-clusters/ece-enable-ccs-for-eck.md

```diff
@@ -8,19 +8,87 @@ applies_to:
+These steps describe how to configure remote clusters between an {{es}} cluster in {{ece}} (ECE) and an {{es}} cluster running within [{{eck}} (ECK)](/deploy-manage/deploy/cloud-on-k8s.md). Once that's done, you'll be able to [run CCS queries from {{es}}](/solutions/search/cross-cluster-search.md) or [set up CCR](/deploy-manage/tools/cross-cluster-replication/set-up-cross-cluster-replication.md).
 
+:::{include} _snippets/terminology.md
+:::
 
-## Establish trust between two clusters [ece_establish_trust_between_two_clusters]
 
-The first step is to establish trust between the two clusters.
+## Allow the remote connection [ece_allow_the_remote_connection_4]
 
+:::{include} _snippets/allow-connection-intro.md
+:::
 
-### Establish trust in the {{ece}} cluster [ece_establish_trust_in_the_elastic_cloud_enterprise_cluster]
+:::::::{tab-set}
+
+::::::{tab-item} API key
+
+:::{include} _snippets/apikeys-intro.md
+:::
+
+### Prerequisites and limitations [ece_prerequisites_and_limitations_4]
+
+:::{include} _snippets/apikeys-prerequisites-limitations.md
+:::
+
+### Enable the remote cluster server interface on the remote ECK cluster
+
+:::{include} _snippets/eck_rcs_enable.md
+:::
+
+### Configure external access to the remote cluster server interface
+
+:::{include} _snippets/eck_rcs_expose.md
+:::
+
+
+### Retrieve the ECK-managed CA certificate of the remote cluster server [fetch-ca-cert]
+
+:::{include} _snippets/eck_rcs_retrieve_ca.md
+:::
+
+### Create a cross-cluster API key on the remote cluster [ece_create_a_cross_cluster_api_key_on_the_remote_deployment_4]
+
+:::{include} _snippets/apikeys-create-key.md
+:::
+
+
+### Configure the local deployment [ece_configure_the_local_deployment_2]
+
+:::{include} _snippets/apikeys-local-config-intro.md
+:::
+
+The steps to follow depend on whether the certificate authority (CA) presented by the remote cluster server, proxy, or load-balancing infrastructure is publicly trusted or private.
+
+::::{dropdown} The CA is public
+
+:::{include} _snippets/apikeys-local-ece-remote-public.md
+:::
+
+::::
+
+::::{dropdown} The CA is private (ECK-managed transport certificates)
+
+When adding the CA certificate in the next steps, use either the ECK-managed transport CA obtained [previously](#fetch-ca-cert), or the CA of the component that terminates TLS connections to clients.
+
+:::{include} _snippets/apikeys-local-ece-remote-private.md
+:::
+::::
+
+::::::
+
+::::::{tab-item} TLS certificate (deprecated)
+### Establish mutual trust between the clusters [ece_establish_trust_between_two_clusters]
+
+When using TLS certificates-based authentication, the first step is to establish trust between the two clusters, by adding the CA certificate and trust details of each environment into the other.
+
+#### Establish trust in the {{ece}} cluster [ece_establish_trust_in_the_elastic_cloud_enterprise_cluster]
 
 1. Save the ECK CA certificate to a file. For a cluster named `quickstart`, run:
```

Comment:
this is old - not sure if you were planning on changing it - but this pattern of linking deeply into the other tutorial is not ideal. I realize this is the deprecated approach so it's not as important to tidy up, but just stating this for the record.